scripts/watch-link.js

```diff
@@ -0,0 +1,38 @@
+const chokidar = require("chokidar")
+const { debounce } = require("lodash")
+const { spawn } = require("child_process")
+
+const run = (cmd, args = []) =>
+  new Promise((resolve, reject) => {
+    if (!Array.isArray(args)) {
+      args = [args]
+    }
+    const p = spawn(cmd, args, {
+      stdio: [process.stdin, process.stdout, process.stderr],
+      cwd: process.cwd(),
+      env: process.env,
+      shell: true,
+    })
+    p.once("error", err => {
+      reject(err)
+    })
+    p.once("exit", code => {
+      if (code === 0) {
+        resolve()
+      } else {
+        reject(new Error(`Exited with ${code}`))
+      }
+    })
+  })
+
+const updateDevPackage = () => {
+  run("yarn", "yalc publish --force --changed")
```

Comment:
If you were to pass these as an array of `args` then you should be able to get rid of the need for an intermediate shell. (Won’t make much of a difference here, so more just a general tip for when it matters.)